nibabel/gifti/tests/test_parse_gifti_fast.py

```diff
@@ -462,3 +462,11 @@ def test_external_file_failure_cases():
     parser = GiftiImageParser()
     with pytest.raises(GiftiParseError):
         img = parser.parse(xmldata)
+
+
+def test_load_compressed():
+    for ext in ('', '.gz', '.bz2'):
+        fn = pjoin(IO_DATA_PATH, 'external.gii' + ext)
+        img7 = load(fn)
```

Comment:
I think you need to create `external.gii.gz` and `external.gii.bz2` in https://github.com/nipy/nibabel/tree/master/nibabel/gifti/tests/data.